README.md

```diff
@@ -180,12 +180,14 @@ Maven snippet:
 #### Requirements
 
 * JDK 8 or later
-* [Apache Maven](https://maven.apache.org/) version 3.5.2 or later
-
-To build, use the command:
 
+To build on Linux/MacOS X use:
+```
+./mvnw clean package -DskipTests
+```
+for Windows use:
 ```
-mvn clean package -DskipTests
+mvnw.cmd clean package -DskipTests
```

Comment:
If you are sending a PR, you would want to run checkstyle and findbugs though

Reply:
@viliam-durina Good one, I completely forgot you could omit a file extension on Windows! 

Do Windows accept the "dot-slash" if you are executing a script from a current directory? If they do then we could re-unify Windows and Unix-like examples (as they were). 
So instead of

_To build on Linux/MacOS X use:_
```
./mvnw clean package -DskipTests
```
_for Windows use:_
```
mvn clean package -DskipTests	mvnw clean package -DskipTests
```

We could have just:

_To build, use the command:_
```
./mvnw clean package -DskipTests
```

Reply:
It would have to be `.\mvnw` though :)

Reply:
>If you are sending a PR, you would want to run checkstyle and findbugs though

Reply:
>If you are sending a PR, you would want to run checkstyle and findbugs though
The PR builder will do it, it will also run tests... I guess most people will build in order to get a package to test their fix. That's what I need package for mostly.